azure-pipelines.yml

```diff
@@ -77,3 +77,27 @@ jobs:
   condition: and(succeeded(), variables.task)
   steps:
   - template: ci/build-single-steps.yml
+
+- job: buildSharedNpm_win
```

Comment:
Not sure the answer but, do we need / want to build these all the time?

Reply:
Probably not - I wanted to keep this pr as simple as possible though since any changes are liable to get lost in the sea of files being migrated over. For PRs I'd like to only do it when there are changes.

Reply:
Can we create an issue to track this work so we don't lose it?

Reply:
https://github.com/microsoft/azure-pipelines-tasks/issues/13520